packages/f-checkout/test/specs/component/f-checkout.component.spec.js

```diff
@@ -12,7 +12,7 @@ describe('f-checkout component tests', () => {
         expect(CheckoutComponent.isAllergenLinkDisplayed()).toBe(true);
     });
 
-    forEach(Object.keys(CheckoutComponent.inputs)).it('should display all fields', key => {
+    forEach(Object.keys(CheckoutComponent.inputs)).it.skip('should display all fields', key => {
```

Comment:
My bad, just added these to save time when running locally, will un-skip!

The 'submit checkout form' test has been skipped for awhile though because it's not quite finished - might wait until your PR is merged before I tackle that one, want to see how you've structure the consts in the page object 👍